src/test/java/longtimenosee/logic/parser/FindCommandParserTest.java

```diff
@@ -71,17 +71,17 @@ public void parse_invalidNameArgs_throwsParseException() {
     public void parse_validPhoneArgs_returnsFindCommand() {
         // 8 digit number
         FindCommand expectedFindCommand =
-                new FindCommand(List.of(new PhoneMatchesNumberPredicate("87654321")));
+                new FindCommand(List.of(new PhoneContainsNumberPredicate("87654321")));
```

Comment:
Just a minor nitpick, maybe we could test for something closer to the max boundary of 25 digit numbers (maybe a 24 digit number?)